lib/internal/Magento/Framework/MessageQueue/ConnectionTypeResolver.php

```diff
@@ -32,15 +32,18 @@ public function getConnectionType($connectionName)
     {
         $type = null;
 
+if(is_array($this->resolvers)){
         foreach ($this->resolvers as $resolver) {
             $type = $resolver->getConnectionType($connectionName);
             if ($type != null) {
                 break;
             }
+}
         }
 
+
         if ($type === null) {
-            throw new \LogicException('Unknown connection name ' . $connectionName);
+            throw new \LogicException('Unknown connection name ' . $connectionName . ' check if Magento_Amqp module enabled');
```

Comment:
Please revert those changes.
Here can be theoretically any resolver enabled (depends from available integrations), not only Amqp.

Reply:
@nuzil - fixed, makes sense.